tbx/blog/tests/test_models.py

```diff
@@ -15,45 +14,8 @@ def test_create(self):
 class TestBlogPageFactory(WagtailPageTestCase):
     def test_create(self):
         blog_post = BlogPageFactory()
-        self.assertEqual(blog_post.related_services.count(), 1)
+        self.assertEqual(blog_post.related_services.count(), 0)
```

Comment:
Also check `related_sectors`

```suggestion
        self.assertEqual(blog_post.related_services.count(), 0)
        self.assertEqual(blog_post.related_sectors.count(), 0)
```